channels/us.m3u

```diff
@@ -25,21 +25,21 @@ https://30a-tv.com/feeds/xodglobal/30atv.m3u8
 https://30a-tv.com/my999.m3u8
 #EXTINF:-1 tvg-id="3ABN.us" tvg-name="3ABN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/GVZuKc7.png" group-title="Religious",3ABN
 https://moiptvhls-i.akamaihd.net/hls/live/652042/secure/master.m3u8
-#EXTINF:-1 tvg-id="3ABNDaretoDream.us" tvg-name="3ABN Dare to Dream" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zI6jr9L.png" group-title="Religious",3ABN Dare to Dream
+#EXTINF:-1 tvg-id="K17JID3.us" tvg-name="3ABN Dare to Dream" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zI6jr9L.png" group-title="Religious",3ABN Dare to Dream
 https://moiptvhls-i.akamaihd.net/hls/live/652313/secure/master.m3u8
 #EXTINF:-1 tvg-id="3ABNFrancais.us" tvg-name="3ABN Français" tvg-country="US" tvg-language="French" tvg-logo="https://i.imgur.com/V6bL5MJ.png" group-title="Religious",3ABN Français
 https://moiptvhls-i.akamaihd.net/hls/live/652314/secure/master.m3u8
 #EXTINF:-1 tvg-id="3ABNInternational.us" tvg-name="3ABN International" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/vhb8zvB.png" group-title="Religious",3ABN International
 https://moiptvhls-i.akamaihd.net/hls/live/652312/secure/master.m3u8
 #EXTINF:-1 tvg-id="3ABNKids.us" tvg-name="3ABN Kids" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/eOvzf3d.png" group-title="Religious",3ABN Kids
 https://moiptvhls-i.akamaihd.net/hls/live/652318/secure/master.m3u8
-#EXTINF:-1 tvg-id="3ABNLatino.us" tvg-name="3ABN Latino" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/4ydVVIA.png" group-title="Religious",3ABN Latino
+#EXTINF:-1 tvg-id="K17JID4.us" tvg-name="3ABN Latino" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/4ydVVIA.png" group-title="Religious",3ABN Latino
 http://uni5rtmp.tulix.tv:1935/bettervida/bettervida/playlist.m3u8
-#EXTINF:-1 tvg-id="3ABNLatino.us" tvg-name="3ABN Latino" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/4ydVVIA.png" group-title="Religious",3ABN Latino
+#EXTINF:-1 tvg-id="K18JLD2.us" tvg-name="3ABN Latino" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/4ydVVIA.png" group-title="Religious",3ABN Latino
```

Comment:
Same programmation than the one above, but different call signs ?

Reply:
When streaming, some of them were showing different content. So, I'm trying to match them to the EPG. All have different call signs.